fluent-langneg/data/likelySubtags-min.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is so small that I think we should include this in the main package.  Can rollup bundle JSON files? If not, I'd just put this in `src/subtags.js`.